src/js/xhr.js

```diff
@@ -1,5 +1,6 @@
-import * as VjsUtils from './util';
```

Comment:
We totally should just get rid of this file and import `xhr` or another similar module from npm.